src/zeit/push/browser/tests/test_form.py

```diff
@@ -10,6 +12,14 @@ class SocialFormTest(zeit.cms.testing.BrowserTestCase):
 
     def setUp(self):
         super(SocialFormTest, self).setUp()
+        # Add payload_template placehodler
+        # all other aad browser tests should fail now
+        with zeit.cms.testing.site(self.getRootFolder()):
+            with zeit.cms.testing.interaction():
+                self.repository['data'] = Folder()
+                self.repository['data']['payload-templates'] = Folder()
+                self.repository['data']['payload-templates']['foo.json'] = \
```

Comment:
Will das nicht evtl. lieber in einen Layer (z.B. [so wie hier](https://github.com/ZeitOnline/zeit.content.dynamicfolder/blob/master/src/zeit/content/dynamicfolder/testing.py#L32))?